mocks/output.json

```diff
@@ -0,0 +1,226 @@
+{
+	"blockHeight": 914117,
+	"timestamp": 1650043442070,
+	"PSTHolders": {
+		"ka_hAAWru0qzK0KprCbvHX5J6QJtkxEiDOHMM1kNBs8": 100
+	},
+	"wallets": {
+		"ka_hAAWru0qzK0KprCbvHX5J6QJtkxEiDOHMM1kNBs8": {
+			"daily": {
+				"byteCount": 1000000,
+				"changeInPercentage": {
+					"24h": -50,
+					"7d": 40
+				},
+				"fileCount": 10,
+				"rankPosition": 49,
+				"tokensEarned": 100
+			},
+			"weekly": {
+				"byteCount": 100,
+				"changeInPercentage": {
+					"24h": -50,
+					"7d": 40
+				},
+				"fileCount": 10,
+				"rankPosition": 49,
+				"tokensEarned": 100
+			},
+			"total": {
+				"byteCount": 100,
+				"changeInPercentage": {
+					"24h": -50,
+					"7d": 40
+				},
```

Comment:
I'm currently not tracking the change in percentage of the total. But I think we don't need it :thinking:

Reply:
https://github.com/ardriveapp/inferno-leaderboard/pull/2/commits/033d2bf4d4e81b03a8f56ec6fdda9d9f220205eb